src/main/java/at/ac/tuwien/kr/alpha/grounder/parser/ParseTreeVisitor.java

```diff
@@ -434,11 +454,7 @@ public ComparisonOperator visitBinop(ASPCore2Parser.BinopContext ctx) {
 	@Override
 	public ComparisonAtom visitBuiltin_atom(ASPCore2Parser.Builtin_atomContext ctx) {
 		// builtin_atom : term binop term;
-		return new ComparisonAtom(
-			(Term) visit(ctx.term(0)),
-			(Term) visit(ctx.term(1)),
-			visitBinop(ctx.binop())
-		);
+		return new ComparisonAtom((Term) visit(ctx.term(0)), (Term) visit(ctx.term(1)), visitBinop(ctx.binop()));
```

Comment:
I think the old formatting was done on purpose.

Reply:
fixed